docs/tutorials/time-dependent-operators.md

```diff
@@ -173,9 +173,9 @@ Array([[0.+0.j, 1.+0.j],
 ??? Note "Function with additional arguments"
     To define a modulated time-array with additional arguments, you can use the optional `args` parameter of [`dq.modulated()`][dynamiqs.modulated]:
     ```python
-    f = lambda t, omega: jnp.cos(omega * t)
+    f = lambda t: jnp.cos(omegas * t)
     omega = 1.0
-    H = dq.modulated(f, dq.sigmax(), args=(omega,))
+    H = dq.modulated(f, dq.sigmax())
```

Comment:
The first line is now wrong. I guess we can explain how to use a `functools.partial` for optional arguments in this short paragraph.